### DISCUSSION TITLE
Architecture Support Request: aarch64

### DISCUSSION THREAD

**Discussion by RavHilton** [crazy-max/docker-cloudflared#100] [Q&A] [answered]
I'm looking at running Pihole, your cloudflared and a number of other docker containers on my ROCK Pi 4 setup here, but it seems that you don't (yet) support the required "aarch64" architecture.

Any chance you could please add that to your project?

**Comment by crazy-max** [comment, marked as answer]
@RavHilton,

> support the required "aarch64" architecture.

This platform is already available (`aarch64 ~ linux/arm64`). Try `docker pull crazymax/cloudflared`.

**Comment by RavHilton** [comment]
Woohoo.  I *had a feeling* they were the same, but wasn't sure.  Thanks for a) replying so quickly and b) helping me learn something.  :)

Looking forward to getting this setup running smoothly - I have been using regular Raspberry Pi units before, but the ability to add SSDs and/or NVMe devices to the ROCK Pi 4 gives me greater scope...

(And as to craziness, I tried sanity for a little while and it didn't suit me, either.)  :)
